### DISCUSSION TITLE
SSH Tunnel--when selecting key file browser cannot browse to hidden folders

### DISCUSSION THREAD

**Discussion by averell-l3l** [dbeaver/dbeaver#22515] [Q&A] [answered]
### Description

This is the clone of this issue [https://github.com/dbeaver/dbeaver/issues/1152](https://github.com/dbeaver/dbeaver/issues/1152) as I'm not the original owner to re-open it.

DBeaver 23.3.2.202401072024
MacOS Sonoma 14.1.1

===========================
The file browser cannot browse to ~/.ssh at all. All ".xxx" directories are hidden. To work, the path to the key must be entered manually.

Suggested easy fix is to start out in that directory...

Using LinuxMint 18.1, and assuming similar issues in ubuntu 16.04 -- if not, my apologies and ticket can be ignored.

### DBeaver Version

DBeaver 23.3.2.202401072024

### Operating System

MacOS Sonoma 14.1.1

### Database and driver

PostgreSQL
Doesn't matter

### Steps to reproduce

1. Create a new connection with SSH tunnel
2. Try to browser a private key file in a hidden folder (like `~/.ssh/`)

**Comment by flambchap** [comment, marked as answer]
Hi. Finder doesn't have a UI toggle to show hidden files when used as a file picker. You can observe the same behavior when opening files or folders in other apps. To bypass this, you can use the `Shift + ⌘ + .` shortcut to show/hide hidden files.
